webroot/css/scss/layout/_nav.scss

```diff
@@ -297,14 +297,15 @@ nav {
             min-width: 0;
             word-wrap: break-word;
             background-color: white;
-            border-radius: 10px;
             margin-bottom: 1.5rem;
             box-shadow: 0 20px 27px 0 rgb(0 0 0 / 5%);
```

Comment:
can we encapsulate?

Reply:
Variables will be used to avoid using pixels